javascript/javascript3/week3/shopping-cart/main.js

```diff
@@ -0,0 +1,118 @@
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+
+  convertToCurrency(currency) {
+    fetch(`https://api.exchangerate-api.com/v4/latest/DKK`)
+      .then((response) => response.json())
+      .then((data) => {
+        let currencyConvert = (data.rates[currency] * this.price).toFixed(2);
+        console.log(
```

Comment:
usually you want to do a return here and console log outside the function (where you're calling it from)